tripy/tripy/frontend/module/groupnorm.py

```diff
@@ -70,15 +70,17 @@ def __init__(self, num_groups: int, num_channels: int, dtype: datatype.dtype = d
             group_norm.weight = tp.ones_like(group_norm.weight)
             group_norm.bias = tp.zeros_like(group_norm.bias)
 
-            input = tp.iota((1, 4, 2, 2))
+            input = tp.iota((1, 4, 2, 2), dim=1)
```

Comment:
Looks like this is dependent on your other PR. Can you move this to draft mode until that's merged?